appengine/standard_python37/django/mysite/settings.py

```diff
@@ -0,0 +1,150 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 2.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'lldtg$9(wi49j_hpv8nnqlh!cj7kmbwq0$rj7vy(b(b30vlyzj'
```

Comment:
Do our other Django samples hardcode a secret key like this? It seems like we should have a replace-me placeholder or something, but you can follow the lead of our other Django samples if needed.

Reply:
Yeah, our App Engine Python 2.7 Django sample uses the sample pattern as well. I will add a comment to remind developers of changing the secret key.

Reply:
Added. Weirdly the snapshot won't refresh :(